src/rgw/rgw_process.cc

```diff
@@ -182,6 +195,56 @@ int process_request(RGWRados* const store,
     goto done;
   }
 
+  // Ask OPA for a decision
+  char buff[100];
+  snprintf(buff, sizeof(buff), "{\"input\":{\"method\":\"%s\", \"user\":\"%s\"}}", s->info.env->get("REQUEST_METHOD"), s->user->display_name.c_str());
```

Comment:
snprintf?  in new code, I think we prefer boost::format or even sstream over this